.mk/cluster.mk

```diff
@@ -35,3 +35,8 @@ kind: kind-cleanup kind-setup istio-setup istio-status
 
 .PHONY: minikube
 minikube: minikube-cleanup minikube-setup istio-setup istio-status
+
+.PHONY: install-cert-local
+install-cert-local:
+	sudo cp hack/tools/registry/ca.crt /usr/local/share/ca-certificates
+	sudo update-ca-certificates
```

Comment:
Is there a way to update certificates without having sudo permissions on a machine?

Reply:
I did not find any on ubuntu. OS X seems to have such a utility.

Reply:
having `root` access on the dev machine is a reasonable requirement, I suggest not spending time to bypass this, @shlomitk1  I would suggest using a local dev environment (on your laptop) rather than a shared account on an IT provisioned VM